src/librustc_lint/builtin.rs

```diff
@@ -227,6 +227,12 @@ impl UnsafeCode {
 }
 
 impl<'a, 'tcx> LateLintPass<'a, 'tcx> for UnsafeCode {
+    fn check_attribute(&mut self, cx: &LateContext, attr: &ast::Attribute) {
+        if attr.check_name("allow_internal_unsafe") {
+            self.report_unsafe(cx, attr.span, "`allow_internal_unsafe` allows defining macros using unsafe without triggering the `unsafe_code` lint at their call site");
```

Comment:
This line is too long (100 chars maximum line length)

Reply:
I separated it on different lines.